eng/doc/DeveloperGuide.md

```diff
@@ -138,6 +138,47 @@ You can use your build of `go` in VS Code by following these steps:
 1. Open the command palette.
 1. Search for `Developer: Reload Window` and select it.
 
-## Making changes to go/src
+## Making Changes to `go/src`
```

Comment:
We don't need to follow the Microsoft style guide, but it does say to use sentence case here rather than title case, and this doc and other docs in the repo already do. https://learn.microsoft.com/en-us/style-guide/scannable-content/headings#formatting-headings